### PR TITLE
User CSV import

### PR DESCRIPTION
**Pull request by threedaymonk**
This gives us a basic user model with a manager relationship and the ability to import users from a CSV file.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class User < ActiveRecord::Base
+  include EmailNormalization
+
+  belongs_to :manager, class_name: :User
+  has_many :employees, class_name: :User, foreign_key: :manager_id
+
+  validates :email, presence: true, format: /\A.+@.+\z/, uniqueness: true
```
Comment (jasiek) [inline app/models/user.rb:7]: Typo?

Comment (threedaymonk) [inline app/models/user.rb:7]: @jasiek Minimal viable validation. Check that the email goes in the right field. Everything else is waste, as email address parsing is complex, and the only way to be sure is to send an email.

Comment (jasiek) [inline app/models/user.rb:7]: Oh, I meant `\Z` instead of `\z` at the end.

Comment (threedaymonk) [inline app/models/user.rb:7]: That's [a Ruby peculiarity](http://ruby-doc.org/core-2.2.0/doc/regexp_rdoc.html):

> - \Z - Matches end of string. If string ends with a newline, it matches just before newline
> - \z - Matches end of string

Comment (jasiek) [inline app/models/user.rb:7]: :+1:

File: features/step_definitions/import_users.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+require 'open3'
+
+Given 'there are no users in the database' do
+  # NOOP
+end
+
+Given 'there are users in the database' do
+  User.create!(name: 'Alixe', email: 'alice@example.com')
```
Comment (novotnyjakub) [inline features/step_definitions/import_users.rb:8]: I'm adding FactoryGirl for the objectives features, but we can unify that later.

Comment (threedaymonk) [inline features/step_definitions/import_users.rb:8]: Yes, I expected that we'd change those in due course.

**Pull request merged**